<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>ItemModifyierDialog</class>
	<widget class="QDialog" name="ItemModifyierDialog">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>450</width>
				<height>520</height>
			</rect>
		</property>
		<property name="minimumSize">
			<size>
				<width>450</width>
				<height>520</height>
			</size>
		</property>
		<property name="maximumSize">
			<size>
				<width>450</width>
				<height>520</height>
			</size>
		</property>
		<property name="windowTitle">
			<string>Éditeur d'élément</string>
		</property>
		<property name="modal">
			<bool>true</bool>
		</property>
		<layout class="QVBoxLayout" name="verticalLayout">
			<property name="spacing">
				<number>15</number>
			</property>
			<property name="leftMargin">
				<number>20</number>
			</property>
			<property name="topMargin">
				<number>20</number>
			</property>
			<property name="rightMargin">
				<number>20</number>
			</property>
			<property name="bottomMargin">
				<number>20</number>
			</property>
			<item>
				<widget class="QLabel" name="titleLabel">
					<property name="font">
						<font>
							<pointsize>11</pointsize>
							<bold>true</bold>
						</font>
					</property>
					<property name="text">
						<string>Informations de l'élément</string>
					</property>
					<property name="alignment">
						<set>Qt::AlignmentFlag::AlignCenter</set>
					</property>
				</widget>
			</item>
			<item>
				<layout class="QFormLayout" name="formLayout">
					<property name="labelAlignment">
						<set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
					</property>
					<property name="horizontalSpacing">
						<number>15</number>
					</property>
					<property name="verticalSpacing">
						<number>12</number>
					</property>
					<item row="0" column="0">
						<widget class="QLabel" name="nameLabel">
							<property name="text">
								<string>Nom de l'élément :</string>
							</property>
						</widget>
					</item>
					<item row="0" column="1">
						<widget class="QLineEdit" name="nameLineEdit">
							<property name="placeholderText">
								<string>Ex: Canapé, Table, Lit...</string>
							</property>
						</widget>
					</item>
					<item row="1" column="0">
						<widget class="QLabel" name="quantityLabel">
							<property name="text">
								<string>Quantité :</string>
							</property>
						</widget>
					</item>
					<item row="1" column="1">
						<widget class="QSpinBox" name="quantitySpinBox">
							<property name="minimum">
								<number>1</number>
							</property>
							<property name="maximum">
								<number>999</number>
							</property>
							<property name="value">
								<number>1</number>
							</property>
						</widget>
					</item>
					<item row="2" column="0">
						<widget class="QLabel" name="volumeLabel">
							<property name="text">
								<string>Volume unitaire (m³) :</string>
							</property>
						</widget>
					</item>
					<item row="2" column="1">
						<widget class="QDoubleSpinBox" name="volumeDoubleSpinBox">
							<property name="decimals">
								<number>2</number>
							</property>
							<property name="minimum">
								<double>0.010000000000000</double>
							</property>
							<property name="maximum">
								<double>999.990000000000009</double>
							</property>
							<property name="singleStep">
								<double>0.100000000000000</double>
							</property>
							<property name="value">
								<double>1.000000000000000</double>
							</property>
						</widget>
					</item>
					<item row="3" column="0">
						<widget class="QLabel" name="areaLabel">
							<property name="text">
								<string>Pièce :</string>
							</property>
						</widget>
					</item>
					<item row="3" column="1">
						<layout class="QHBoxLayout" name="areaHBoxLayout">
							<property name="spacing">
								<number>10</number>
							</property>
							<item>
								<widget class="QComboBox" name="areaComboBox">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Expanding" vsizetype="Fixed">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="placeholderText">
										<string>Sélectionner une pièce...</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="addAreaButton">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>80</width>
											<height>0</height>
										</size>
									</property>
									<property name="text">
										<string>Ajouter</string>
									</property>
								</widget>
							</item>
						</layout>
					</item>
					<item row="4" column="0" colspan="2">
						<widget class="Line" name="line">
							<property name="frameShadow">
								<enum>QFrame::Shadow::Plain</enum>
							</property>
							<property name="orientation">
								<enum>Qt::Orientation::Horizontal</enum>
							</property>
						</widget>
					</item>
					<item row="5" column="0">
						<widget class="QLabel" name="demontageLabel">
							<property name="text">
								<string>Démontage :</string>
							</property>
						</widget>
					</item>
					<item row="5" column="1">
						<widget class="QCheckBox" name="demontageCheckBox">
							<property name="text">
								<string>Nécessite un démontage</string>
							</property>
						</widget>
					</item>
					<item row="6" column="0">
						<widget class="QLabel" name="remontageLabel">
							<property name="text">
								<string>Remontage :</string>
							</property>
						</widget>
					</item>
					<item row="6" column="1">
						<widget class="QCheckBox" name="remontageCheckBox">
							<property name="text">
								<string>Nécessite un remontage</string>
							</property>
						</widget>
					</item>
					<item row="7" column="0">
						<widget class="QLabel" name="heavyLabel">
							<property name="text">
								<string>Objet lourd :</string>
							</property>
						</widget>
					</item>
					<item row="7" column="1">
						<widget class="QCheckBox" name="heavyCheckBox">
							<property name="text">
								<string>Objet lourd ou encombrant</string>
							</property>
						</widget>
					</item>
					<item row="8" column="0" colspan="2">
						<widget class="Line" name="line_2">
							<property name="frameShadow">
								<enum>QFrame::Shadow::Plain</enum>
							</property>
							<property name="orientation">
								<enum>Qt::Orientation::Horizontal</enum>
							</property>
						</widget>
					</item>
					<item row="9" column="0">
						<widget class="QLabel" name="noteLabel">
							<property name="text">
								<string>Note :</string>
							</property>
							<property name="alignment">
								<set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
							</property>
						</widget>
					</item>
					<item row="9" column="1">
						<widget class="QTextEdit" name="noteTextEdit">
							<property name="maximumSize">
								<size>
									<width>16777215</width>
									<height>80</height>
								</size>
							</property>
							<property name="placeholderText">
								<string>Informations complémentaires (couleur, matériau, état...)</string>
							</property>
						</widget>
					</item>
				</layout>
			</item>
			<item>
				<spacer name="verticalSpacer">
					<property name="orientation">
						<enum>Qt::Orientation::Vertical</enum>
					</property>
					<property name="sizeHint" stdset="0">
						<size>
							<width>20</width>
							<height>10</height>
						</size>
					</property>
				</spacer>
			</item>
			<item>
				<widget class="QDialogButtonBox" name="buttonBox">
					<property name="orientation">
						<enum>Qt::Orientation::Horizontal</enum>
					</property>
					<property name="standardButtons">
						<set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
					</property>
					<property name="centerButtons">
						<bool>false</bool>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections>
		<connection>
			<sender>buttonBox</sender>
			<signal>accepted()</signal>
			<receiver>ItemModifyierDialog</receiver>
			<slot>accept()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>20</x>
					<y>20</y>
				</hint>
				<hint type="destinationlabel">
					<x>20</x>
					<y>20</y>
				</hint>
			</hints>
		</connection>
		<connection>
			<sender>buttonBox</sender>
			<signal>rejected()</signal>
			<receiver>ItemModifyierDialog</receiver>
			<slot>reject()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>20</x>
					<y>20</y>
				</hint>
				<hint type="destinationlabel">
					<x>20</x>
					<y>20</y>
				</hint>
			</hints>
		</connection>
	</connections>
</ui>